MetaCocktailsSwiftData/Model/Cocktails/Classics/PeguClub.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PeguClub.swift
+//  MetaCocktailsSwiftData
+//
+//  Created by James Menkal on 8/17/24.
+//
+
+import Foundation
+
+var peguClub = Cocktail(cocktailName: "Pegu Club Cocktail",
+                        glasswareType: .stemmedGlassware,
```

Comment:
I think we should remove the "cocktail" since these are all cocktails. It can be called a pegu club without being confused for the bar. Same with the "Casanova Cocktail" - that can just be "Casanova"

Reply:
That's how the cocktails were originally printed. So I think they should remain that way.